plugins/meta/flannel/flannel.go

```diff
@@ -241,6 +247,109 @@ func cmdAdd(args *skel.CmdArgs) error {
 	return delegateAdd(args.ContainerID, n.DataDir, n.Delegate)
 }
 
+func cmdAddWindows(containerID string, n *NetConf, fenv *subnetEnv) error {
+
+	n.Delegate["name"] = n.Name
+
+	if !hasKey(n.Delegate, "type") {
+		n.Delegate["type"] = "wincni.exe"
+	}
+
+	updateOutboundNat(n.Delegate, fenv)
+
+	n.Delegate["cniVersion"] = "0.2.0"
```

Comment:
If possible, can you use `0.3.1`? It supports a more advanced return type.

Reply:
Updated to use existing (linux) logic: 
n.Delegate["cniVersion"] = n.CNIVersion